extra-i18n/fcitx5-chinese-addons/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=fcitx5-chinese-addons
+PKGDES="Fcitx 5 Chinese addons"
+PKGDEP="curl libime qt-5 fcitx5-qt boost opencc fcitx5 gettext"
```

Comment:
```suggestion
PKGDEP="curl libime fcitx5-qt boost opencc gettext"
```

Reply:
why remove fcitx5?

Reply:
and curl can be remove?

Reply:
ahh...ok, cURL looks like the cloud Pinyin plugin needs

Reply:
>why remove fcitx5?

You already included `fcitx5-qt` which implies the inclusion of `qt-5` and `fcitx5`.

Reply:
> > why remove fcitx5?
> 
> You already included `fcitx5-qt` which implies the inclusion of `qt-5` and `fcitx5`.

ok